nucypher/blockchain/eth/sol/source/contracts/Issuer.sol

```diff
@@ -168,6 +171,16 @@ contract Issuer is Upgradeable {
         currentSupply2 = currentSupply2 - _amount;
     }
 
+    /**
+    * @notice Burn sender's tokens. Amount of tokens will be returned for future minting
+    * @param _value Amount to burn
+    */
+    function burn(uint256 _value) public {
```

Comment:
Is it really "burn"? Perhaps a term like `giveBack` is more correct.

Reply:
not necessary back, because you can "burn" even tokens that you have with airdrop